src/clients/cassandra/CassandraConnector.js

```diff
@@ -65,13 +62,14 @@ function executeBatchMutations(mutations) {
  */
 function executeQuery(query, params) { // eslint-disable-line no-unused-vars
```

Comment:
Paging done with `pageSize` parameter in [9f40f2b](https://github.com/CatalystCode/project-fortis-services/pull/77/commits/9f40f2be88c7aad928595fa35c6373004f00f221)